requests/download_tpdi_deliveries.py

```diff
@@ -0,0 +1,102 @@
+from oauthlib.oauth2 import BackendApplicationClient
```

Comment:
I think it would be good to add a comment on the top here, giving a short description of what the script does.

To start giving everything here more structure we could also add a Readme for this directory, where you can also add the comment